docs: reformat the documentation (#396)
ansys/ansys-sphinx-theme: doc/source/user-guide/autoapi.rst

.. _ref_user_guide_autoapi:

> Autoapi extension
> ==================

Reviewer: ```suggestion
Sphinx AutoAPI
===============
```

> To use the ``autoapi`` along with the ``ansys-sphinx-theme``, you need to

Reviewer: ```suggestion
To use Sphinx AutoAPI with the Ansys Sphinx Theme, you must
```

> add ``ansys_sphinx_theme.extension.autoapi`` to the ``extensions`` list in your ``conf.py`` file

Reviewer: ```suggestion
add ``ansys_sphinx_theme.extension.autoapi`` to the ``extensions`` variable in your Sphinx ``conf.py`` file
```

Author: sure, that sounds good, thanks @PipKat
